### PR TITLE
exploit the notifications api to increase display time

### PR DESCRIPTION
**Pull request by psimyn**
add Create and Destroy for longer lasting notification displays

### CODE REVIEW

File: js/background.js [hunk 1]
```diff
@@ -32,8 +33,38 @@ function NotificationsController($scope, $http) {
...
+            // setup 30 minute timer
+            if (typeof _this.notifications[url].dismissed === 'undefined') {
+                setTimeout(function() {
+                    _this.destroyNotification(url);
+                }, 30 * 60 * 1000);
+            }
+            if (!_this.notifications[url].dismissed) {
+                _this.notifications[url].dismissed = false;
+                _this.notifications[url].callback = setTimeout(function() {
+                    options.priority = options.priority == 2 ? 1 : 2;
```
Comment (GVRV) [inline js/background.js]: What's the reason behind this?

File: js/background.js [hunk 2]
```diff
@@ -32,8 +33,38 @@ function NotificationsController($scope, $http) {
...
+                    _this.createNotification(url, options);
```
Comment (GVRV) [inline js/background.js]: Why are we calling `createNotification` every second?

**Comment by GVRV** [comment]
Code is much simpler than earlier solution, and works as intended! Good to go! :+1:

**Comment by geeknam** [comment]
back to basic :(

**Pull request merged**